config/initializers/line_item_controller_gift_card_details.rb

```diff
@@ -0,0 +1,3 @@
+Rails.application.config.to_prepare do
+  Spree::Api::LineItemsController.line_item_options += [:gift_card_details]
```

Comment:
Is that a situation that we need to worry about at this point? If that becomes a requirement down the line maybe we can circle back then?